Library/Formula/fdm.rb

```diff
@@ -0,0 +1,21 @@
+class Fdm < Formula
+  homepage "http://sourceforge.net/projects/fdm/"
+  url "https://downloads.sourceforge.net/project/fdm/fdm/fdm-1.8/fdm-1.8.tar.gz"
+  sha1 "25c0ebc0c2b43984d04c82d0ba3909acf7403df7"
+
+  depends_on "tdb"
```

Comment:
Is `tdb` used by much/any software other than `fdm`? If not, we should probably install it as a `resource` of the `fdm` formula.